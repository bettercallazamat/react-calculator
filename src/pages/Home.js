const Home = () => (
  <div className="homeDiv">
    <h1>Welcome to my Calculator Project</h1>
    <p>
      This project is part of my learning React framework with the aim of building a Calculator
    </p>
    <p className="list-title">Features List:</p>
    <ul>
      <li>Add</li>
      <li>Subtract</li>
      <li>Multiply</li>
      <li>Divide</li>
      <li>Turn number into percentage</li>
      <li>Change sign of a numbers</li>
      <li>Use float numbers</li>
    </ul>
    <p className="list-title">Built With:</p>
    <ul>
      <li>Node.js</li>
      <li>React</li>
      <li>React-DOM</li>
      <li>React-Create-App</li>
      <li>React-Router-Dom</li>
      <li>React Hooks</li>
      <li>npm</li>
      <li>Heroku</li>
    </ul>
  </div>
);

export default Home;
