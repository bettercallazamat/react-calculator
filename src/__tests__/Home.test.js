import React from 'react';
import { render } from '@testing-library/react';
import Home from '../pages/Home';

describe('Snapshot Home page', () => {
  it('should match the snapshot', () => {
    const { asFragment } = render(<Home />);
    expect(asFragment(<Home />)).toMatchSnapshot();
  });
});
