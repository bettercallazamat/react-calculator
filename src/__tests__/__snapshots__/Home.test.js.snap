// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Home page should match the snapshot 1`] = `
<DocumentFragment>
  <div
    class="homeDiv"
  >
    <h1>
      Welcome to my Calculator Project
    </h1>
    <p>
      This project is part of my learning React framework with the aim of building a Calculator
    </p>
    <p
      class="list-title"
    >
      Features List:
    </p>
    <ul>
      <li>
        Add
      </li>
      <li>
        Subtract
      </li>
      <li>
        Multiply
      </li>
      <li>
        Divide
      </li>
      <li>
        Turn number into percentage
      </li>
      <li>
        Change sign of a numbers
      </li>
      <li>
        Use float numbers
      </li>
    </ul>
    <p
      class="list-title"
    >
      Built With:
    </p>
    <ul>
      <li>
        Node.js
      </li>
      <li>
        React
      </li>
      <li>
        React-DOM
      </li>
      <li>
        React-Create-App
      </li>
      <li>
        React-Router-Dom
      </li>
      <li>
        React Hooks
      </li>
      <li>
        npm
      </li>
      <li>
        Heroku
      </li>
    </ul>
  </div>
</DocumentFragment>
`;
